src/domain/community/virtualContributor/vcSettingsPage/VCSettingsPage.tsx

```diff
@@ -13,11 +17,31 @@ export const VCSettingsPage = () => {
     },
   });
 
+  const [updateContributorMutation] = useUpdateVirtualContributorMutation();
+
+  const handleUpdate = async virtualContributor => {
```

Comment:
I would just call the function here, there's no need to await if you're not doing anything after